python/python/glide/async_commands/redis_modules/search/commands.py

```diff
@@ -0,0 +1,96 @@
+        See https://redis.io/commands/ft.create/ for more details.
+
+        Args:
+            client (TRedisClient): The Redis client to execute the command.
+            index (Index): The definition of the index, including its name and type.
+            schema (List[Field]): The schema of the index, specifying the fields and their types.
+            max_text_fields (bool): Forces RediSearch to encode indexes as if there were more than 32 text attributes.
+                Additional attributes (beyond 32) can be added using FT.ALTER.
+            temporary_seconds (Optional[int]): Creates a lightweight temporary index that automatically expires after a set period of inactivity, measured in seconds.
+                The index's internal idle timer resets each time a search or addition operation occurs.
+                Because such indexes are lightweight, you can create thousands of such indexes without negative performance implications and, therefore,
+                you should consider using SKIPINITIALSCAN to avoid costly scanning.
+                Warning: When temporary indexes expire, they drop all the records associated with them.
+            offset (Offset): Whether to store term offsets for documents.
+            highlight (Highlights): Whether to disable / enable highlighting support.
+            field_flags (FieldFlags): Whether to store attribute bits for each term.
+            frequencies (Frequencies): Whether to save the term frequencies in the index.
+            stop_words (Optional[List[str]]): Sets the index with a custom stopword list, to be ignored during indexing and search time.
+                If not set, FT.CREATE takes the default list of stopwords.
+            initial_scan (InitialScan): Whether to scan and index.
+
+        Returns:
+            A simple OK response.
+
+        Examples:
+            >>> index = Index("my_index", IndexType.HASH)
+            >>> schema = [TextField("name"), NumericField("age")]
+            >>> await search_commands.create_index(
+                    redis_client,
+                    index,
+                    schema,
+                    max_text_fields=True,
+                    temporary_seconds=3600,
+                    field_flags=FieldFlags.NO_FIELDS,
+                    stopwords=["the", "and", "is"]
+                )
+                "OK"
+        """
+        args = index.get_index_atr()
+        if max_text_fields:
+            args.append("MAXTEXTFIELDS")
+        if temporary_seconds is not None:
+            args.extend(["TEMPORARY", str(temporary_seconds)])
+        if offset == Offset.NO_OFFSET:
+            args.append("NOOFFSETS")
+        if highlight == Highlights.NO_HIGHLIGHTS:
+            args.append("NOHL")
+        if field_flags == FieldFlags.NO_FIELDS:
+            args.append("NOFIELDS")
+        if frequencies == Frequencies.NO_FREQUENCIES:
+            args.append("NOFREQS")
+        if stop_words:
```

Comment:
stop_words can be an empty list to disable all stop words, so we should get into this 'if' with an empty list, however the boolean value of an empty list is false. so you should do:
```
if stop_words is not None:

Reply:
please add a test with empty list to verify that it works